README.md

```diff
@@ -519,6 +521,11 @@ These settings can be changed at any time.
   Bottlerocket enables the `json-file`, `awslogs`, and `none` drivers by default.
 * `settings.ecs.loglevel`: The level of verbosity for the ECS agent's logs.
   Supported values are `debug`, `info`, `warn`, `error`, and `crit`, and the default is `info`.
+* `settings.ecs.metadata-service-rps`: The steady state rate limit of the throttling configurations set for the task metadata service
+* `settings.ecs.metadata-service-burst`: The burst rate limit of the throttling configurations set for the task metadata service
+* `settings.ecs.reserved-memory`: The amount of memory, in MB, reserved for critical system processes
```

Comment:
It's really only possible to reserve 64 GiB of memory? On the one hand that seems like a lot, but on the other hand there are EC2 instance types with quite a lot more memory.

Reply:
Well, the only thing that is setting "a limit" is the type of the field:

```golang
// ReservedMemory specifies Reduction, in MiB, of the memory capacity of the instance
// that is reported to Amazon ECS. Used by Amazon ECS when placing tasks on container instances.
// This doesn't reserve memory usage on the instance
ReservedMemory uint16
```

So yes, only 64Gib even in hosts with hundreds of Gib.